cli/context/store/store.go

```diff
@@ -12,12 +12,15 @@ import (
 	"encoding/json"
 	"io"
 	"net/http"
+	"os"
 	"path"
 	"path/filepath"
 	"regexp"
 	"strings"
 
 	"github.com/docker/docker/errdefs"
+	"github.com/gofrs/flock"
+	"github.com/hashicorp/go-multierror"
```

Comment:
Can we use Go's native multi-errors for this? (`errors.Join()`), or would we loose too much functionality?

I think the general trend is to (try) moving away from hashicorp's packages as licensing for some of them becomes more risky (plus trying to use stdlib where possible in general)

Reply:
I can switch to `errors.Join` if you don't mind requiring Go 1.20. The vendor.mod requires Go 1.19 at the moment.

Reply:
I've pushed a few patches to switch to the Go 1.20 errors package to see how it looks.